specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1565,6 +1942,143 @@
       },
       "description": "Represents metrics values."
     },
+    "PercentileMetricListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PercentileMetric"
+          },
+          "description": "The list of percentile metrics for the account."
+        }
+      },
+      "description": "The response to a list percentile metrics request."
+    },
+    "PercentileMetric": {
+      "properties": {
+        "startTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the metric (ISO-8601 format)."
+        },
+        "endTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The end time for the metric (ISO-8601 format)."
+        },
+        "timeGrain": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The time grain to be used to summarize the metric values."
+        },
+        "unit": {
+          "$ref": "#/definitions/UnitType",
+          "description": "The unit of the metric."
+        },
+        "name": {
+          "readOnly": true,
+          "$ref": "#/definitions/MetricName",
+          "description": "The name information for the metric."
+        },
+        "metricValues": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PercentileMetricValue"
+          },
+          "description": "The percentile metric values for the specified time window and timestep."
+        }
+      },
+      "description": "Percentile Metric data"
+    },
+    "PercentileMetricValue": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/MetricValue"
+        }
+      ],
+      "properties": {
+        "P10": {
```

Comment:
Properties should follow camelCase format. For example, it should be `p10` instead of `P10`. It applies for all properties, `P10`, `P25`, ...